ui/contexts/snaps/snap-interface.tsx

```diff
@@ -102,22 +111,21 @@ export const SnapInterfaceContextProvider: FunctionComponent<
         params: {
           event: {
             type: event,
-            // TODO: Allow null in the types and simplify this
-            ...(name !== undefined && name !== null ? { name } : {}),
```

Comment:
Is this not a problem anymore? As far as I can tell we might be including an undefined `value` or `name` now 🤔

Reply:
Is this a type issue or runtime issue? There are no type errors at least

Reply:
It was a runtime issue at one point where we were sending `undefined` and crashing